### ISSUE TITLE
windows 2019 nodes are not registering with k8s v1.22.3-rancher1-1 in an RKE1 cluster

### ISSUE THREAD

**Issue by anupama2501** [rancher/rancher#35614]
**Rancher Server Setup**
- Rancher version: `v2.6.3-rc1`
- Installation option (Docker install/Helm Chart): Docker install
   - If Helm Chart, Kubernetes Info: k3s
         Cluster Type (RKE1, RKE2, k3s, EKS, etc):
         Node Setup: 1 node 
         Version: `v1.22.3+k3s1`
- Proxy/Cert Details: `self-signed`

**Information about the Cluster**
- Kubernetes version: v1.22.3
- Cluster Type (Local/Downstream): `Downstream infrastructure provider 3 worker, 1 etcd, 1 cp`

**Describe the bug**
Windows 2019 nodes are not registering with the cluster that has k8s v1.22.3-rancher1-1. Therefore, the nodes are stuck in resgistering.This is happening as the k8s 1.22 version is not in the code yet. 
https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.6/rke/k8s_windows_service_options.go#L8

**To Reproduce**
1. Create an rke1 custom cluster with 1 etcd, 1 cp and 1 worker linux nodes
2. Nodes register fine
3. Register windows 2019 nodes in the cluster.
 
**Result**
Nodes are stuck in registering and won't come up active

**Expected Result**
Expected the nodes to register and come up active with the k8s 1.22

**Additional context**
Errors 
```
2021/11/19 19:40:26 [ERROR] error syncing 'c-f779h/p-b7ns5': handler system-image-upgrade-controller: upgrade cluster c-f779h system service logging failed: template system-library-rancher-logging incompatible with rancher version or cluster's [c-f779h] kubernetes version, requeuing
2021/11/19 19:40:58 [ERROR] error syncing 'c-hmhfq/m-6f8e4cebb0da': handler rke-worker-upgrader: getNodePlan error for node [m-6f8e4cebb0da]: getDefaultKubernetesServicesOptions: No serviceOptions found for cluster version [v1.22.3-rancher1-1] or cluster major version [v1.22], requeuing
2021/11/19 19:41:12 [ERROR] error syncing 'c-hsvl5/m-e409a635d34b': handler rke-worker-upgrader: getNodePlan error for node [m-e409a635d34b]: getDefaultKubernetesServicesOptions: No serviceOptions found for cluster version [v1.22.3-rancher1-1] or cluster major version [v1.22], requeuing
```

**Comment by phillipsj** [comment]
@sowmyav27 this has been updated here: https://releases.rancher.com/kontainer-driver-metadata/dev-v2.6-1.22/data.json

**Comment by kinarashah** [comment]
Available to test with v2.6-head once https://drone-publish.rancher.io/rancher/rancher/6914 passes

**Comment by sowmyav27** [comment]
**On 2.6-head commit id: `e322f8f`**

- Deployed windows RKE1 custom cluster on k8s 1.22.3-rancher1-1
- Nodes registered successfully.
- Validation tests look good on the cluster
